packages/docs/src/pages/en/getting-started/installation.md

```diff
@@ -273,6 +273,38 @@ const app = createApp()
 app.use(vuetify).mount('#app')
 ```
 
+## Using as ES Module with CDN
+
+To import Vuetify (and Vue) using an [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap) you can use the same CDN but contain it in a ES module without tooling
+
+```html
+<head>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vuetify@{{ version }}/dist/vuetify.min.css" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css" />
+  <link rel="stylesheet" href="https://fonts.bunny.net/css?family=roboto:400,500,700" />
+  <script type="importmap">
+  {
+    "imports": {
+      "vue": "https://cdn.jsdelivr.net/npm/vue@{{ version }}/dist/vue.esm-browser.js",
```

Comment:
version is the vuetify version

Reply:
should be fine to just do `@latest`, also is on line 292